<ui version="4.0" >
 <class>DownloadMapDialog</class>
 <widget class="QDialog" name="DownloadMapDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Download</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="Username" />
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="Website" />
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Website</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="Password" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="IncludeTracks" >
     <property name="text" >
      <string>Also download raw GPS tracks</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Location</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QRadioButton" name="FromBookmark" >
          <property name="text" >
           <string>Bookmark</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Bookmarks" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>7</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="FromView" >
        <property name="text" >
         <string>Current view</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QRadioButton" name="FromviewAndAdd" >
          <property name="text" >
           <string>Current view and add as bookmark</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="NewBookmark" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>383</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DownloadMapDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>223</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel" >
     <x>241</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DownloadMapDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>279</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel" >
     <x>283</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FromBookmark</sender>
   <signal>toggled(bool)</signal>
   <receiver>Bookmarks</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>59</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel" >
     <x>129</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FromviewAndAdd</sender>
   <signal>toggled(bool)</signal>
   <receiver>NewBookmark</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>99</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel" >
     <x>361</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
